CHANGELOG.rst

```diff
@@ -17,20 +17,154 @@ Changes to "0.14.0"
+  - ``READ_LINEAR``: For reading through ADIOS2 steps, for full support of ADIOS2 BP5 #1291 #1379
+- Support for all char types (CHAR SCHAR UCHAR) #1275 #1378
+- Header for openPMD-defined error types #1080 #1355
+- Support for ADIOS2 BP5 engine #1119 #1215 #1258 #1262 #1291
+- Support for selecting flush targets (buffer/disk) in ADIOS2 BP5 for more fine-grained memory control #1226 #1207
+- Add file extensions for ADIOS2: ``.bp4``, ``.bp5`` and furthers, make them behave more as expected #1218
+- Add ``Series::close()`` API call #1324
+- Optionally explicitly map ADIOS2 steps to openPMD iterations via modifiable attributes (only supported in experimental ADIOS2 modes) #949
+- Support for ADIOS 2.8 and newer #1166
+- I/O optimizations for HDF5 #1129 #1133 #1192
+- Support for empty string attributes #1087 #1223 #1338
+- Support for inconsistent padding of filenames in file-based encoding #1118 #1253
+- ADIOS2: Automatic (de)activation of span API depending on compression configuration #1155
+- ADIOS2: Support for operator specification at read time #1191
+- Support for array specializations of C++ smart pointer types #1296
+- Direct support for raw pointer types in ``store/loadChunk()`` API, replacing former ``shareRaw()`` #1229
+- Support and backend optimizations (ADIOS2 BP5) based on unique pointer types in ``store/loadChunk()`` #1294
+- Use C++ ``std::optional`` types in public Attribute API (``Attribute::getOptional<T>()``) for dynamic attribute type conversion #1278
 
 Bug Fixes
 """""""""
 
+- HDF5
+
+  - Support attribute reads from HDF5 Vlen Strings #1084
+  - Close HFD5 handles in availableChunks task #1386
+- ADIOS1
+  - Fix use-after-free issue #1224
+
+- ADIOS2
+
+  - Don't apply compression operators multiple times #1152
+  - Fix logic for associating openPMD objects to files and paths therein #1073
+  - Fix precedence of environment variable vs. JSON configuration
+  - Detect changing datatypes and warn/fail accordingly #1356
 - CMake:
 
   - MPI: prefer HDF5 in Config package, too #1340
+  - ADIOS1: do not include as -isystem #1076
+  - Fix ``-Wsign-compare`` #1202
+  - Remove caching of global CMake variables #1313
+  - Fix Build & Install Option Names #1326
+  - Prefer parallel HDF5 in find_package in downstream use #1340
+  - CMake: Multi-Config Generator #1384
+Python:
+
+  - Fix ``__repr__`` (time and Iteration) #1242 #1149
+  - Python Tests: Fix ``long`` Numpy Type #1348
+  - use ``double`` as standard for attributes #1290 #1369kk
+  - Fix dtype_from_numpy #1357
+- Don't forget closing unmodified files #1083
+- Diverse relaxations on attribute type conversions #1085 #1096 #1137
+- Performance bug: Don't reread iterations that are already parsed #1089
+- Performance bug: Don't flush prematurely #1264
+- Avoid object slicing in Series class #1107
+- Avoid use-after-free in Python bindings #1225
+- Logical fixes for opening iterations #1239
+
+Breaking Changes
+""""""""""""""""
+
```

Comment:
Maybe to `NEWS.rst`

Reply:
I've added a detailed upgrade description there, but left this short list in the changelog.